fil_fungible_token/src/token/mod.rs

```diff
@@ -13,27 +15,31 @@ use fvm_ipld_blockstore::Blockstore as IpldStore;
 use fvm_shared::bigint::Zero;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::ActorID;
+use num_traits::Signed;
 
 /// Library functions that implement core FRC-??? standards
 ///
 /// Holds injectable services to access/interface with IPLD/FVM layer.
-pub struct TokenHelper<BS>
+pub struct Token<BS>
```

Comment:
Naming wise went with calling the top-level library that authors will use `Token` 

and the trait/interface that they will implement is defined in `token/types.rs` as `FrcXXXToken`